### PR TITLE
Use default ScriptContext

### PR DESCRIPTION
**Pull request by nini-faroux**
This PR removes all the custom `ScriptContext` types that just mirror the `ScriptContext` type from Plutus, and replaces them in the scripts with `ScriptContext`.

### CODE REVIEW

File: dao/Dao/Vote.hs [hunk 1]
```diff
@@ -185,14 +165,9 @@ data VoteDynamicConfigDatum = VoteDynamicConfigDatum
...
 
-unstableMakeIsData ''VoteMinterAddress
-unstableMakeIsData ''VoteMinterTxOut
-unstableMakeIsData ''VoteMinterTxInInfo
-unstableMakeIsData ''VoteMinterScriptPurpose
-unstableMakeIsData ''VoteMinterScriptContext
-unstableMakeIsData ''VoteMinterTxInfo
 unstableMakeIsData ''VoteMinterDynamicConfigDatum
 
+-- | Redeemer for 'Triphut.Vote.Script.validateVote' validator
```
Comment (Randy1Burrell) [inline dao/Dao/Vote.hs]: This should be `Triphut.Vote.Script.validateVote`

**Review by Randy1Burrell — CHANGES_REQUESTED** [review]
I left a small change here.

File: dao/Dao/Vote.hs (hunk 1 above)
Comment (nini-faroux) [reply dao/Dao/Vote.hs]: Sorry @Randy1Burrell - missed your comment here. Just fixed that now.

**Review by Randy1Burrell — APPROVED** [review]
LGTM! 🥇

**Pull request merged**